language/tools/move-cli/src/lib.rs

```diff
@@ -22,3 +22,344 @@ pub use move_lang::command_line::DEFAULT_OUTPUT_DIR as DEFAULT_BUILD_DIR;
 
 /// Extension for resource and event files, which are in BCS format
 const BCS_EXTENSION: &str = "bcs";
+
```

Comment:
can you leave this in main.rs. It seems unrelated to cleaning up deps

Reply:
Unfortunately it does. Because of native functions I had to introduce a new crate `move-cli-with-df` and a new binary `move-df`. They need to share the same piece CLI code, which I moved from main.rs into lib.rs.